internal/policies/mount/adsys-mount-template.mount

```diff
@@ -0,0 +1,12 @@
+# This template defines the basic structure of a mount unit generated by adsys for system mounts.
+[Unit]
+Description=Mount for %s
+After=network-online.target
+
+[Mount]
+What=%s
+Where=/mnt/adsys/%s
+Options=nofail,%s
```

Comment:
If I understood correctly, there shouldn't be any problems. But I will check again just in case.

Reply:
Reminder: after a discussion, let’s remove all options right now (and in particular nofail), as failing to start the unit:
- will report the error in systemd when it fails
- will only log a warning and not return any error (`systemctl start`) when started or stopped directly from adsys.

Reply:
Yes, I know. As we've talked, we could leave it blank and it would work anyway, however I do feel like it's a good explicit thing to have. This conversation was before me realizing that we would need different approaches based on the type of the mount and that we had to add an option for Kerberos security.